packages/functoria-runtime/functoria-runtime.3.0.1/opam

```diff
@@ -0,0 +1,41 @@
+opam-version: "2.0"
+maintainer:   "Gabriel Radanne <drupyog@zoho.com>"
+authors:      [ "Thomas Gazagnaire"
+                "Anil Madhavapeddy"
+                "Dave Scott"
+                "Thomas Leonard"
+                "Gabriel Radanne" ]
+homepage:     "https://github.com/mirage/functoria"
+bug-reports:  "https://github.com/mirage/functoria/issues"
+dev-repo:     "git+https://github.com/mirage/functoria.git"
+doc:          "https://mirage.github.io/functoria/"
+license:      "ISC"
+tags:         ["org:mirage"]
+
+build: [
+  ["dune" "subst"] {pinned}
+  ["dune" "build" "-p" name "-j" jobs]
+  [make "test"] {with-test}
```

Comment:
```suggestion
  ["dune" "runtest" "-p" name "-j" jobs] {with-test}
  ["env" "INSIDE_FUNCTORIA_TESTS=1" "dune" "exec" -p name "--" "tests/test_full.exe"] {with-test}
```
Could you do something like this instead? Otherwise it might break at a later date due to warnings set as errors by dune using the dev profile

Reply:
It's probably simpler to just remove `make test` then. I'll do that.